server/models/Request.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  user_id: {
+    type: String,
+    required: true
+  },
+  sitter_id: {
+    type: String,
+    required: true
+  },
+  start: {
+    type: Date,
+    required: true,
+    default: Date.now
+  },
+  end: {
+    type: Date,
+    required: true,
+    default: Date.now
+  },
+  accepted: {
+    type: Boolean,
+    default: false
+  },
+  declined: {
+    type: Boolean,
+    default: false
+  },
```

Comment:
I find having only a single field for these (e.g. `status`) with an [enum validator](https://mongoosejs.com/docs/validation.html#built-in-validators) helps simplify the logic later, but I'll leave it up to you to implement this however you think is best 👍🏼